reference/performances_guide/index.rst

```diff
@@ -40,3 +56,192 @@ Here is the complete formula to check if you have, by far, more indexes than the
 .. warning::
 
     If your collection of products requires more than 64 indexes, please contact us (we've an Elastic Search implementation).
+
+More than 10k attributes?
+-------------------------
+
+The number of attributes will impact performances of the PIM in several ways.
+We've tested performances with a set of 10k attributes in total (not 10k attributes per product).
+
+.. warning::
+
+    On this axis, we still have different issues. We've tested with 10k attributes and a low amount of families and categories.
+
+    Screens impacted are the following, we're releasing performances fixes as 1.4 patches:
+     - **[fixed in 1.4.11]** (PIM-5209) the loading of the route /configuration/attribute-group/rest (Community Edition)
+     - **[fixed in 1.4.12] see following chapter** (PIM-5208) the product grid loading due to manage filters design (Community Edition)
+     - **[fixed in 1.4.12] see following chapter** (PIM-5208) the variant product grid loading due to manage filters design (Community Edition)
+     - **[WIP]** the configure step of the product / mass edit / common attributes (Community Edition)
+     - the optional attribute popin in the product edit form (Community Edition)
+     - the configure step of the family / mass edit / requirements (Community Edition)
+     - the edition of a variant group due to large amout of attributes useable as axis (Community Edition)
+
+    **If you plan to use the PIM with more than 10k attributes, please contact us.**
+
+More than 500 attributes usable in the grids?
+---------------------------------------------
+
+The number of attributes used in the grids will impact performances of the PIM in several ways.
+
+Prior to 1.4.12, all attributes could be displayed in the grid columns and only *attributes usable as grid filter*
+could be used to filter data in the grids. During the loading of the product grids, we had a significant performance problem
+for catalogs with a large number of attributes. As we couldn't rework the way the grids are configured in a patch,
+we decided to change this behavior and we're still looking for a way to improve it for 1.5.
+
+From 1.4.12 and upper versions, the attribute option *attributes usable as grid filter* become *usable in grid* . That option will now determine
+if the attribute can be displayed as column or used as a filter in the product grids.
```

Comment:
This option states wheter or not the attribute can be displayed as a comun or used as a filter in the products grids.